queries.sql

```diff
@@ -15,3 +15,86 @@ SELECT * from animals WHERE neutered = TRUE;
 SELECT * from animals WHERE name != 'Gabumon';
 
 SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+/* TRANSACTIONS */
+
+BEGIN;
+
+UPDATE animals
+SET species = 'unspecified';
+
+ROLLBACK;
```

Comment:
- [ ] You are supposed to check if the change was made you can simply do:
 `SELECT species from animals` to verify that change was made
ROLLBACK;

 `SELECT species from animals`; verify that change was undone